index.js

```diff
@@ -208,7 +208,7 @@ class Hyperdiscovery extends EventEmitter {
   }
 
   rejoin (discoveryKey) {
```

Comment:
why is this called 'rejoin' and not join?

Reply:
you have to call `.add(feed)` to join the first time. But then you can leave/rejoin without calling add again.